dotCMS/src/main/java/com/dotcms/rest/api/v1/temp/DotTempFile.java

```diff
@@ -13,31 +15,30 @@ public class DotTempFile {
 
   public final String id,mimeType,referenceUrl,thumbnailUrl,fileName,folder;
   public final boolean image;
+  public final Map<String, Serializable> metadata;
 
   @JsonIgnore
   public final File file;
+
+
+
   
   public DotTempFile(final String id, final File file) {
     super();
     this.id = id;
     this.file = file;
-    this.mimeType= getMimeType(file);
+    this.metadata = canRead(file)? APILocator.getFileStorageAPI().generateRawBasicMetaData(file):null;
```

Comment:
Maybe run this in a `Try.of(()->` to catch IO errors?

Reply:
sounds good